### PR TITLE
Fix import and deprecation issues in unit tests

### PR DESCRIPTION
**Pull request by zengwang430521**

### CODE REVIEW

File: tests/test_model/test_smpl.py [hunk 1]
```diff
@@ -1,10 +1,42 @@
...
 
 from mmpose.models.utils import SMPL
 
 
+def generate_smpl_weight_file(output_dir):
+    """Generate a SMPL model weight file to initialize SMPL model, and generate
+    a 3D joints regressor file."""
+
+    if not os.path.exists(output_dir):
+        os.makedirs(output_dir)
```
Comment (innerlee) [inline tests/test_model/test_smpl.py:16]: Will the generated files be cleaned up after testing?

**Review by innerlee — APPROVED** [review]

File: mmpose/datasets/datasets/body3d/body3d_mpi_inf_3dhp_dataset.py [hunk 2]
```diff
@@ -360,7 +360,7 @@ def _report_3d_auc(self, keypoint_results, mode='3dauc'):
 
         preds = np.stack(preds)
         gts = np.stack(gts)
-        masks = np.ones_like(gts[:, :, 0], dtype=np.bool)
+        masks = np.ones_like(gts[:, :, 0], dtype=bool)
```
Comment (innerlee) [inline mmpose/datasets/datasets/body3d/body3d_mpi_inf_3dhp_dataset.py:363]: why these changes?

Comment (ly015) [reply mmpose/datasets/datasets/body3d/body3d_mpi_inf_3dhp_dataset.py:363]: The original code raises DeprecationWarning with a reminder that changing like this will not change the behavior.
